app/views/correctReturn/SmallProducerDetailsView.scala.html

```diff
@@ -0,0 +1,57 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.correctReturn.routes
+@import views.summary.correctReturn.SmallProducerDetailsSummary
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton,
+    govukSummaryList: GovukSummaryList
+)
+
+@(form: Form[_], mode: Mode, smallProducerList: List[SmallProducer], plural: Boolean)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("correctReturn.smallProducerDetails.title"))) {
+
+    @formHelper(action = routes.SmallProducerDetailsController.onSubmit(mode), Symbol("autoComplete") -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+        @if(plural) {
```

Comment:
Could we not just use `@if(smallProducerList.size == 1)` and remove the need for an extra parameter to be created in the controller - especially as that parameter is totally dependent on one of the other parameters?